compose/foundation/foundation/src/skikoMain/kotlin/androidx/compose/foundation/text/CupertinoTextFieldPointerModifier.skiko.kt

```diff
@@ -175,10 +175,12 @@ private fun getTapHandlerModifier(
 @Composable
 private fun getLongPressHandlerModifier(
     state: TextFieldState,
-    offsetMapping: OffsetMapping
+    offsetMapping: OffsetMapping,
+    manager: TextFieldSelectionManager,
```

Comment:
Redundant comma at the end of the line.

Reply:
It's OK, we usually do that to avoid git diff/conflicts when adding more parameters or array elements